app/containers/message/Video.js

```diff
@@ -37,8 +36,16 @@ const Video = React.memo(({
 		if (isTypeSupported(file.video_type)) {
 			return showAttachment(file);
 		}
-		const uri = formatAttachmentUrl(file.video_url, user.id, user.token, baseUrl);
-		openLink(uri, theme);
+
+		let url = file.title_link || file.video_url;
+		if (!url) {
+			return;
+		}
+		if (!url.startsWith('http')) {
+			url = `${ baseUrl }${ url }`;
+		}
+
+		Navigation.navigate('GenericWebView', { uri: url, headers: { 'x-user-id': user.id, 'x-auth-token': user.token }, title: file.title });
```

Comment:
Both calls are inside `message/Reply` and `message/Video` and they call basically the same screen with composite params.
Can we add a util function for this?